.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,13 @@
+# Declare default permissions as read only.
```

Comment:
The workflow is missing name and trigger.

Reply:
Updated.